### ISSUE TITLE
"Connect Rasa-X to GIT" always appears even when a GIT repo is already added

### ISSUE THREAD

**Issue by mghadam** [RasaHQ/rasa#7197]
**Rasa version**:  2.0.2

**Rasa SDK version**: -

**Rasa X version**: confirmed the bug with v0.33.0 and v0.33.1 docker images

**Python version**: Default Docker version for Rasa 2.0.2 / Rasa x 0.33.0

**Operating system** (windows, osx, ...): Docker version

**Issue**:

I have already connected my rasa-x to git however the “Connect Rasa X to Git” popup always displays and there is no button to go to the next step of instructrions or disable it.

![image](https://user-images.githubusercontent.com/10244293/98331217-5b1b9800-2011-11eb-8043-026c7cfe6180.png)

**Error (including full traceback)**:
```
I don't see any relevant errors in `docker-compose logs | grep -i error`
```

**Command or request that led to error**:
```
Connected git to my rasa x 0.33.0, the helper windows always appears and there is no button going to the next level as shown in the above screenshot
```

**Comment by sara-tagger** [comment]
Thanks for the issue, @melindaloubser1 will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗

**Comment by melindaloubser1** [comment]
Is your git repository still functionally connected when you see this message, or does it get disconnected every time?

**Comment by mghadam** [comment]
It is fully functional, i.e. it restored all my data when I connected Rasa X to git using `curl --request POST --url  "https://MYDOMAIN/api/projects/default/git_repositories?api_token=AUTHKEY" --header 'content-type: application/json' --data-binary @repository.json` and it publishes changes to git whenever I click on `Apply Changes to Git Server` button:

![image](https://user-images.githubusercontent.com/10244293/98764528-406e6800-23f1-11eb-88e3-39902875347b.png)

**Comment by melindaloubser1** [comment]
If you had to make that post request for it to restore your data, then it means it isn't remaining connected to git, you're having to re-connect it each time. When is this happening? Is it after you do a `docker-compose down && docker-compose up -d` or is it happening each time you log in to Rasa X?

**Comment by mghadam** [comment]
> If you had to make that post request for it to restore your data, then it means it isn't remaining connected to git, you're having to re-connect it each time.

Sorry I didn't explain it well. I used the curl command to connect my Rasa X to GIT only one time, just after installing Rasa X for the first time on my server.

> When is this happening? Is it after you do a `docker-compose down && docker-compose up -d` or is it happening each time you log in to Rasa X?

It happens each time I refresh the page, not only after login to Rasa X, but also on each page refresh.

My Rasa X is connected to GIT and it works pretty well, I don't need to reconnect it, it is connected and I can publish changes to the GIT server without any problem.

The problem is that this "Connect Rasa X to GIT" does not have any Next button, and I am not sure why this notification box thinks my Rasa is not connected to GIT while it is connected properly and working.

I can provide you access to the server / platform so you debug it.

**Comment by alwx** [comment]
Hey @mghadam!

Thanks for your detailed report. I was able to reproduce this issue locally but it's hard to say yet what's the reason for that. We will try to check it later this week and prioritize it higher because it seems to be a pretty important issue that might prevent users from finishing the onboarding flow.

**Comment by mghadam** [comment]
I have a similar problem with "Rasa swag" window. I have already filled the form once, however, it appears several times a day. Shall I open a new issue for it?

![screenshot-aifastnet uk nf-2020 11 16-23_53_31](https://user-images.githubusercontent.com/10244293/99304221-52c02a00-2867-11eb-815c-40992cdf59f6.png)

**Comment by MariyaGalasyuk** [comment]
@mjaycub i was not able to reproduce it locally, no popup appears after connecting to git

**Comment by mjaycub** [comment]
@MariyaGalasyuk are you debugging with on-boarding enabled?

**Comment by MariyaGalasyuk** [comment]
@mjaycub so i ran  `SKIP_FRONTEND_BUILD=true RASA_X_PASSWORD=rasa OPEN_WEB_BROWSER=false GIT_DEVELOPMENT_MODE=true rasa x` for backend and `cd src && REACT_APP_API_URL=http://localhost:5002 yarn run develop` for frontend, which in my eyes should enable onboarding steps (?)

**Comment by mjaycub** [comment]
Ok replying in your DMs :D
